pkg/istio/authorizationpolicy/controller_test.go

```diff
@@ -59,7 +59,20 @@ var (
 		},
 	}
 
-	notOnboardedServiceWithAnnotation = &v1.Service{
+	notOnboardedServiceWithAnnotationTrue = &v1.Service{
```

Comment:
is this when roles/policies are not defined for the service but the authz flag in omega.yaml is set to true? If so, can we rename the test to something like `undefinedAthenzRulesServiceWithAnnotationTrue`

Reply:
Yep, updated.